tsconfig.mina-signer.json

```diff
@@ -10,6 +10,11 @@
     "moduleResolution": "nodenext", // comply with node + "type": "module"
     "esModuleInterop": true, // to silence jest
 
+    "experimentalDecorators": true, // needed for decorators
```

Comment:
I kept getting errors related too:

```
../lib/signature.ts:28:9 - error TS1219: Experimental support for decorators is a feature that is subject to change in a future release. Set the 'experimentalDecorators' option in your 'tsconfig' or 'jsconfig' to remove this warning.

28   @prop s: Scalar;
           ~

../lib/signature.ts:28:9 - error TS2564: Property 's' has no initializer and is not definitely assigned in the constructor.

28   @prop s: Scalar;
```
Because of these errors, I had to add these rules to the `tsconfig` file for `mina-signer`. Is this incorrect?

Reply:
yeah I think the problem is that you're now importing `proof_system.ts` (and with it, most of snarkyjs) in `transaction-leaves-bigint.ts` which mina-signer depends on. better to use the verification key hash that's exposed from constants everywhere, not the one from JSOO

Reply:
yep, that was it! I have it changed locally and building works without these changes :+1: